Makefile

```diff
@@ -181,6 +181,10 @@ lint-static-check:
 golint: lint-static-check
 	@$(MAKE) for-all-target TARGET="lint"
 
+.PHONY: gomod-tidy
+gomod-tidy:
```

Comment:
this target name follows the same convention of merging `go` with the command name